prediction_market_agent/agents/microchain_agent/nft_treasury_game/deploy_nft_treasury_game.py

```diff
@@ -71,6 +92,43 @@ def load(self) -> None:
 
         super().load()
 
+    def before_iteration_callback(self) -> None:
```

Comment:
Thanks to this, even if the agent stops playing and the pod gets restarted, it will stop right away again - before doing any actions.